tests/unit/Rules/NumericValTest.php

```diff
@@ -13,66 +13,52 @@
 
 namespace Respect\Validation\Rules;
 
-use PHPUnit\Framework\TestCase;
+use Respect\Validation\Test\RuleTestCase;
 
 /**
  * @group  rule
+ *
  * @covers \Respect\Validation\Exceptions\NumericValException
```

Comment:
This class is not testing `NumericValException` anymore, that's why we have the integration tests